fixtures/schema/02_fbf/01_materialized_views/01_flood_event_roads/08_flood_event_road_district_summary.sql

```diff
@@ -66,7 +66,7 @@ CREATE MATERIALIZED VIEW public.mv_flood_event_road_district_summary AS
         ), flooded_count_selection AS (
          SELECT a.flood_event_id,
             a.district_id,
-            sum(a.road_type_count) AS flooded_flooded_road_count,
+            sum(a.road_type_count) AS flooded_road_count,
```

Comment:
Standardize column name to follow conventions.

Reply:
hi @lucernae it was because that's from the db

Reply:
Yes, I also fix it in the db side, because it's confusing and make the frontend code difficult to understand. The db should not make the frontend job harder.

Reply:
oh okay cool, yeah at that time, I thought it would take a long time to change the db so I just adjusted the front end according to db